src/js/components/FormField/FormField.js

```diff
@@ -250,9 +255,9 @@ const FormField = forwardRef(
           : {};
       contents = (
         <FormFieldContentBox
-          overflow="hidden"
-          {...contentProps}
+          {...fieldContentProps}
           {...innerProps}
+          {...contentProps}
```

Comment:
In terms of order of operations, in this case whatever is specified in `contentProps` will trump all other styling. I am wondering about states where a border or background is specified for errors, disabled, focus, etc. states. This would override the styling for those states, correct? Do we want the styling for those states to be overridden? Alternative approach suggested below...

Also, I was struggling to make sense for what the differences are between `fieldContentProps` and `contentProps`. I think where I landed is that they are effectively the same thing, but originating from two locations (theme, prop).

Should they merge into a single object where we can better account for the manipulation of that object's values? This could help with readability as well as allowing for better control flow of object.

Line 183 could initiate a `adjustedContentProps` with definition from theme. Then apply any modifications specified in `contentProps`, then execute the rest of the object's manipulation based on the existing state based logic. The resulting hierarchy would be: theme.formField.content --> contentProps --> theme.formField.[state].

Reply:
I did think about disabled and such, but I decided I this was the most flexibility. If I'm bothering to specify separate contentProps rather than define it in the theme, this is more of a special case situation and I don't want other things to affect it. If you let disabled and such override what I pass in contentProps then I don't really have a way to do what I want. I'm at the mercy of existing disabled modifiers and such. And I specifically kept contentProps as the passed in user-defined prop. Other props come from the theme for that particular field type. At least that's what was in my brain at the time.

Reply:
Makes sense. I'd need to ponder this a bit more. If we do go the `contentProps` trump all direction, I do think it would be beneficial to explicitly say so / provide guidance for its use in the docs.

Reply:
Added a little more detail in the doc for contentProps